Detectors/TRD/workflow/src/trd-calib-workflow.cxx

```diff
@@ -37,6 +38,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
 {
   auto enableRootInp = configcontext.options().get<bool>("enable-root-input");
+  o2::conf::ConfigurableParam::updateFromString(configcontext.options().get<std::string>("configKeyValues"));
```

Comment:
I only tested the configurable params with the tracking workflow which already called this

Reply:
no problem, happens